tests/api/cart.api.test.js

```diff
@@ -1,41 +1,120 @@
 const { test, expect } = require('@playwright/test')
 
 const { getAuthToken, getProductBody, getProductId, getCartBody, getCartId } = require('../../lib/helpers')
-  
+
 test.describe.parallel('Carts API', () => {
   let authorization
-  
+
   test.beforeAll(async ({ request }) => {
     authorization = await getAuthToken(request)
   })
-  
+
   test('retrieves the list of carts', async ({ request }) => {
     const response = await request.get('/carrinhos')
-    
+
+    await expect(response).toBeOK()
+  })
+
+  test('retrieves an existing cart by its id', async ({ request }) => {
+    const idProduct = await getProductId(request, authorization, getProductBody())
+    const _id = await getCartId(request, authorization, getCartBody(idProduct))
+
+    const response = await request.get(`/carrinhos/${_id}`)
+
     await expect(response).toBeOK()
   })
 
   test('creates a cart successfully', async ({ request }) => {
-    const id = await getProductId(request, authorization, getProductBody())
+    const authorization = await getAuthToken(request)
+    const idProduct = await getProductId(request, authorization, getProductBody())
 
     const response = await request.post('/carrinhos', {
-      data: getCartBody(id),
+      data: getCartBody(idProduct),
       headers: { 'Authorization': authorization }
     })
 
     await expect(response).toBeOK()
 
     const responseBody = JSON.parse(await response.text())
-    
+
     await expect(responseBody.message).toBe('Cadastro realizado com sucesso')
```

Comment:
@uLucasFraga let's not hardcode messages, we don't have access to the codebase. Ideally we would import constants from these messages and use in tests, but it's not the case 😄 
According to the API specification, the only OK scenario would be that one, so it won't be necessary to validate the message.